.buildkite/publish/dra/init_dra_publishing.sh

```diff
@@ -138,7 +138,7 @@ if [[ "${PUBLISH_SNAPSHOT:-}" == "true" ]]; then
   generateDependencyReport $DEPENDENCIES_REPORTS_DIR/$dependencyReportName
 
   echo "-------- Publishing SNAPSHOT DRA Artifacts"
-  cp $RELEASE_DIR/dist/elasticsearch_connectors-${VERSION}.zip $DRA_ARTIFACTS_DIR/connectors-${VERSION}-SNAPSHOT.zip
+  cp $RELEASE_DIR/dist/elasticsearch_connectors-${VERSION}.tar.gz $DRA_ARTIFACTS_DIR/connectors-${VERSION}-SNAPSHOT.tar.gz
```

Comment:
Would be great to confirm that it doesn't break stuff (zip -> tar.gz move). We might need to edit this somewhere, @seanstory might remember this as we've discussed this

Reply:
oh, yes this will break agentless I believe. The agent image currently consumes our zip.
I don't think it's the end of the world, we'd just need to decide if we definitely want tar.gz instead, and if so, we have to update the DRA manifest, and update the agent image build to pick up the new artifact and install it correctly.

Might be better to leave this for later